src/filters/colormatrix/ColorMatrixFilter.js

```diff
@@ -575,6 +577,26 @@ export default class ColorMatrixFilter extends core.Filter
     {
         this.uniforms.m = value;
     }
+
+    /**
+     * The interpolation value to use when mixing the original and resultant colors.
+     *
+     * When the value is 0, the original color is used without modification.
+     * When the value is 1, the result color is used.
+     * When in the range (0, 1) the color is interpolated between the original and result by this amount.
+     *
+     * @member {number}
+     * @default 1
+     */
+    get interpolation()
```

Comment:
Intensity of the effect and the interpolation you use are different values. For example, the `night` effect already has an intensity: `filter.night(intensity, multiply);`.

Setting intensity, then intensity again somewhere else would be weird. `intensity` usually acts as a multiplier increasing or decreasing how strong the affect is, and while interpolation in this case could be seen as similar it is a distinct value.